float64 frameRate
---
bool success